exporter.go

```diff
@@ -157,6 +166,53 @@ func (e *Exporter) Export(opts ExportOptions) (platform.ExportReport, error) {
 	return report, nil
 }
 
+func copySboms(layerDir string, e *Exporter) error {
+	targetBuildDir := filepath.Join(layerDir, "sbom", "build", "buildpacksio_lifecycle")
+	targetLaunchDir := filepath.Join(layerDir, "sbom", "launch", "buildpacksio_lifecycle")
+
+	if err := os.MkdirAll(targetBuildDir, os.ModePerm); err != nil {
+		return errors.Wrapf(err, "fail to create directory %v", targetBuildDir)
+	}
+
+	if err := os.MkdirAll(targetLaunchDir, os.ModePerm); err != nil {
+		return errors.Wrapf(err, "fail to create directory %v", targetLaunchDir)
+	}
+
+	extensions := [3]string{".sbom.cdx.json", ".sbom.spdx.json", ".sbom.syft.json"}
+	components := [2]string{"lifecycle", "launcher"}
+
+	var targetDir string
+	for _, component := range components {
+		if component == "lifecycle" {
+			targetDir = targetBuildDir
+		} else if component == "launcher" {
+			targetDir = targetLaunchDir
+		}
+		for _, extension := range extensions {
+			sbomFilename := component + extension
+
+			srcSbomAbsPath := "/cnb/lifecycle/" + sbomFilename
+
+			_, err := os.Stat(srcSbomAbsPath)
+			if !errors.Is(err, os.ErrNotExist) {
+				destSbomFilename := strings.Replace(sbomFilename, component+".", "", 1)
```

Comment:
Would "sbom" + extension be simpler?

Reply:
destSbomFilename := strings.Replace(sbomFilename, component+".", "", 1)

==> no, because we want to change the name of the file: lifecycle.sbom.cdx.json ==> sbom.cdx.json and the value of the _extension_ variable is ".sbom.cdx.json"